examples/ImageJImgLib2.ipynb

```diff
@@ -37,8 +37,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Initialize imagej. Point to a local Fiji installation\n",
-    "# Replace the value below with to your local Fiji installation.\n",
+    "# Initialize imagej.\n",
+    "# Replace the value below to your local Fiji installation.\n",
```

Comment:
what about  `Replace the value below with the path to your local Fiji installation`

Reply:
@jcfr yes, that is right -- thanks!